src/App.js

```diff
@@ -388,19 +385,15 @@ function App() {
 
         <div className="flowSection">
           <row>
-            <span>
-              <div className="instructions">
-                <p>Select a
-                </p>
-                <span>
-                  {!selectedArtist ? <HeaderComponent placeholder="cantor" /> : <HeaderComponent placeholder="season" />}
-                </span>
-              </div>
-            </span>
-
-            <span>
+          <span>
               {selectedArtist && (
-                <div className="instructions">
+                <div className="instructions" onClick={() => {
+                  setSelectedArtist();
+                  setSelectedSeason();
+                  setTabIndex(0)
+                  setCantorContainerStyle({})
+                  setTextStyle({})
+                }}>
```

Comment:
to get the return to work you have to set the state to how it was before that page was opened. To return to cantor tab you have to empty the artist and season states and reset the tab and styles